project-metrics/README.md

```diff
@@ -0,0 +1,41 @@
+AMP Project Metrics
+===================
+
+This GitHub App reports metrics for the repository and scores each metric on a
+common scale.
+
+
+Components
+----------
+
+This app is composed of two parts. The first is a Google App Engine (GAE)
+application in Python which collects data, computes metrics, and provides a REST
+API interface to retrieve them. The other is a GitHub app which queries the REST
+API to display the project metrics for a repository.
+
+
+Local development
+-----------------
+
+### Setup
+
+Follow these setup instructions to start developing for the GAE backend locally:
+
+1. Clone this repository and cd into the `project_metrics/metrics_service`
+   directory
+2. Create and activate a Python 3 [virtualenv](virtualenv.pypa.io/en/latest/)
+3. `pip install flask` to install the flask runtime
```

Comment:
Can't this be part of requirements.txt then?

Reply:
All requirements actually need to be installed once in the virtualenv to run locally and once in `third_party` to deploy to App Engine. Modified README accordingly.